### PR TITLE
06_Valid_Parentheses

### PR DESCRIPTION
**Pull request by Naveenkenshi**

### CODE REVIEW

File: Naveenraj/06_Valid_Parentheses/main.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import "fmt"
+
+func main() {
+	input := ""
+	fmt.Println(Valid_Parentheses(input))
+}
+
+func Valid_Parentheses(input string) bool {
+	listOfBrackets := map[string]string{
```
Comment (kenshilabstech) [inline Naveenraj/06_Valid_Parentheses/main.go:13]: Take input from console

File: Naveenraj/06_Valid_Parentheses/main.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		return false
+	}
+
+	if len(input)%2 != 0 {
+		return false
+	}
+
+	// first method - failed in test case - {[]}
+
+	// for i := 0; i < len(input); i = i + 2 {
```
Comment (kenshilabstech) [inline Naveenraj/06_Valid_Parentheses/main.go]: Remove commented out code

**Review by kenshilabstech — CHANGES_REQUESTED** [review]

File: Naveenraj/06_Valid_Parentheses/main.go (hunk 1 above)
Comment (Naveenkenshi) [reply Naveenraj/06_Valid_Parentheses/main.go:13]: Now taken inputs from console

File: Naveenraj/06_Valid_Parentheses/main.go (hunk 2 above)
Comment (Naveenkenshi) [reply Naveenraj/06_Valid_Parentheses/main.go]: removed the commented code

**Pull request merged**